homeassistant/components/lifx/button.py

```diff
@@ -0,0 +1,77 @@
+"""Button entity for LIFX devices.."""
+from __future__ import annotations
+
+from homeassistant.components.button import (
+    ButtonDeviceClass,
+    ButtonEntity,
+    ButtonEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, IDENTIFY, RESTART
+from .coordinator import LIFXUpdateCoordinator
+from .entity import LIFXEntity
+
+RESTART_BUTTON_DESCRIPTION = ButtonEntityDescription(
+    key=RESTART,
+    name="Restart",
+    device_class=ButtonDeviceClass.RESTART,
+    entity_registry_enabled_default=False,
+    entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
Buttons should use `EntityCategory.CONFIG`.

Reply:
Fix in #76788